microservice/package/src/main/resources/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine:3
```

Comment:
could we use 'FROM arm64v8/alpine:3.14' which has wide support of various architectures?